cluster/addons/cluster-loadbalancing/glbc/glbc.yaml

```diff
@@ -1,3 +1,8 @@
+{% set kube_uid = "" -%}
```

Comment:
@andyzheng0831 This file is getting templated in preparation for running as a static pod on the master. FYI.